### PR TITLE
 Fix and simplify reactlog's version check approach.

### LINKED ISSUE #4011: reactlog doesn't work with latest shiny
```r
> library(shiny)
> library(reactlog)
> 
> # tell shiny to log all reactivity
> reactlog_enable()
> 
> # run a shiny app
> app <- system.file("examples/01_hello", package = "shiny")
> runApp(app)

Listening on http://127.0.0.1:6145
^C
Run `rlang::last_trace()` to see where the error occurred.

Run `rlang::last_trace()` to see where the error occurred.
> shiny::reactlogShow()
Error: invalid version specification ‘1.6), markdown, rmarkdown, ggplot2, reactlog (>= 1.0.0’
```

### PR DESCRIPTION
**Pull request by cpsievert**
Closes #4011

### CODE REVIEW

File: R/graph.R [hunk 1]
```diff
@@ -110,34 +82,15 @@ renderReactlog <- function(sessionToken = NULL, time = TRUE) {
...
-    version
+check_reactlog <- function() {
+  if (!is_installed("reactlog", reactlog_version)) {
+    rlang::check_installed("reactlog", reactlog_version)
   }
-})
+}
 
+# Should match the (suggested) version in DESCRIPTION file
+reactlog_version <- "1.0.0"
```
Comment (schloerke) [inline R/graph.R:93]:
```suggestion
reactlog_min_version <- "1.0.0"
```

**Review by schloerke — APPROVED** [review]
Thank you!

Only suggestion is for a long variable name.

File: tests/testthat/test-reactlog.R [hunk 2]
```diff
@@ -126,3 +126,16 @@ test_that("message logger appears", {
   })
 
 })
+
+
+test_that("reactlog_version is as expected", {
+  suggests <- strsplit(packageDescription("shiny")$Suggests, "\n")[[1]]
```
Comment (cpsievert) [inline tests/testthat/test-reactlog.R:132]:
```suggestion
  suggests <- strsplit(packageDescription("shiny")$Suggests, ",")[[1]]
```

File: tests/testthat/test-reactlog.R [hunk 3]
```diff
@@ -126,3 +126,16 @@ test_that("message logger appears", {
...
+
+test_that("reactlog_version is as expected", {
+  suggests <- strsplit(packageDescription("shiny")$Suggests, ",")[[1]]
+  reactlog <- trimws(
+    grep("reactlog", suggests, value = TRUE)
+  )
+  expect_length(reactlog, 1)
+  expect_equal(
+    reactlog,
+    sprintf("reactlog (>= %s),", reactlog_min_version)
```
Comment (cpsievert) [inline tests/testthat/test-reactlog.R:139]:
```suggestion
    sprintf("reactlog (>= %s)", reactlog_min_version)
```

File: DESCRIPTION [hunk 4]
```diff
@@ -210,4 +210,5 @@ Roxygen: list(markdown = TRUE)
 RdMacros: lifecycle
 Config/testthat/edition: 3
 Config/Needs/check:
-  shinytest2
+  shinytest2,
+  rprojroot
```
Comment (cpsievert) [inline DESCRIPTION:214]:
```suggestion
  shinytest2
```

**Pull request merged**